uk.co.studiopond.spiderbro2.yml

```diff
@@ -0,0 +1,30 @@
+app-id: uk.co.studiopond.spiderbro2
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: spiderbro2_linux
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+modules:
+  - shared-modules/glu/glu-9.json
+  - name: Spiderbro-2
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 spiderbro2_linux /app/bin/spiderbro2_linux
+      - install -Dm755 polybius2_linux /app/bin/polybius2_linux
+      - install -Dm755 polybius_linux /app/bin/polybius_linux
+      - install -Dm755 vameforspiderbroforspiderbro2_linux /app/bin/vameforspiderbroforspiderbro2_linux
+      - cp -r data /app/bin/data
+      - unzip -o polydata2.zip -d /app/bin
+      - unzip -o polydata.zip -d /app/bin
+      - unzip -o spiderbrodata.zip -d /app/bin
```

Comment:
Having some proper releases on GitHub would be better, serving binary files and archives through a git repo is a bit iffy.

Reply:
So should we wait for releases?

Reply:
Apologies for the delay. I have added releases here https://github.com/pforpond/Spiderbro-2 I hope this is satisfactory.

Reply:
Can you also switch the manifest to use those releases please?